pkg/sqs/SqsConsumer.php

```diff
@@ -185,6 +185,10 @@ protected function convertMessage(array $sqsMessage): SqsMessage
     {
         $message = $this->context->createMessage();
 
+        if (isset($sqsMessage['MessageId'])) {
```

Comment:
No need for if. As far as I can see the field is there all the time.

Reply:
@makasim ok, i've update my code to remove the isset check and have also move the setMessage call to the end of the function, because the setHeader is overriding the message_id header value

Reply:
The `if` is still there, isn't it? Should be removed as the `MessageId` is always there.

Reply:
oups, my bad i was sure to have pushed the deletion on this `if`